app/assets/main/react_components/dashboard/climate_actions/manage_actions/kanban/KanbanActionColumn.jsx

```diff
@@ -1,17 +1,18 @@
 import React from "react";
 import { Droppable } from "react-beautiful-dnd";
-import KanbanActionItem from "./KanbanActionItem.jsx";
+import Card from "./card_components/Card.jsx";
```

Comment:
Maybe something more specific than card, like KanbanCard - since there will also be CarouselCards.